src/binding.cc

```diff
@@ -2581,7 +2625,7 @@ v8::Isolate* v8__internal__GetIsolateFromHeapObject(const v8::Data& data) {
              : nullptr;
 }
 
-int v8__internal__Object__GetHash(const v8::Data& data) {
+int v8__internal__Object__GetHash(const v8::Value& data) {
```

Comment:
A bit confusing, function name says `Object` but it's `v8::Value` passed. Can you add a comment here?

Reply:
I've renamed the function to `v8__Value__GetHash`.
PS: it's true that that V8 uses different class names than they do in the external API.
`i::Object` corresponds most closely to `Data`, but this particular function works only for true javascript values.